pkgs/by-name/sw/swayfx-unwrapped/package.nix

```diff
@@ -29,18 +32,6 @@
     sha256 = "sha256-Gwewb0yDVhEBrefSSGDf1hLtpWcntzifPCPJQhqLqI0=";
   };
 
-  # This patch was backported into SwayFX
-  # remove when next release is rebased on Sway 1.9
-  patches =
-    let
-      removePatches = [
-        "LIBINPUT_CONFIG_ACCEL_PROFILE_CUSTOM.patch"
-      ];
-    in
-    builtins.filter
-      (patch: !builtins.elem (patch.name or null) removePatches)
-      (oldAttrs.patches or [ ]);
```

Comment:
I think this is another good example on why trying to reuse code sometimes ends up unnecessary complicating things.

I am in favor of just making those 2 derivations completely independent. WDYT @raboof ?

Reply:
It's a balance between avoiding duplication and being autonomous.

For my own Sway fork, Volare, the build 'itself' is independent (https://codeberg.org/raboof/volare/src/branch/main/flake.nix#L10) but I do re-use the sway module (https://codeberg.org/raboof/volare#when-using-flakes-for-your-nixos-configuration).

Reply:
I think this approach is just causing an additional maintenance burden. Decoupling would solve all problems and avoid new ones in the future.

Also, if I remember correctly overriding packages causes a few strange issues, for example the update scripts don't work as it should (they try to update the original package instead the overriden one).

Reply:
> Also, if I remember correctly overriding packages causes a few strange issues, for example the update scripts don't work as it should

I would call this a bug in the update script. But yes, let's eventually decouple these.